src/confluent_kafka/src/Admin.c

```diff
@@ -976,6 +976,537 @@ static PyObject *Admin_alter_configs (Handle *self, PyObject *args,
+        size_t i;
+        PyObject *result;
+
+        result = PyList_New(c_acls_cnt);
+
+        for (i = 0 ; i < c_acls_cnt ; i++) {
+                PyList_SET_ITEM(result, i,
+                        Admin_c_AclBinding_to_py(c_acls[i]));
+        }
+
+        return result;
+}
+
+
+/**
+ * @brief Convert C acl_result_t array to py list.
+ */
+static PyObject *
+Admin_c_acl_result_to_py (const rd_kafka_acl_result_t **c_result,
+                            size_t cnt) {
+        PyObject *result;
+        size_t ti;
+
+        result = PyList_New(cnt);
+
+        for (ti = 0 ; ti < cnt ; ti++) {
+                PyObject *error;
+                const rd_kafka_error_t *c_error = rd_kafka_acl_result_error(c_result[ti]);
+
+                error = KafkaError_new_or_None(
+                        rd_kafka_error_code(c_error),
+                        rd_kafka_error_name(c_error));
+
+                PyList_SET_ITEM(result, ti, error);
+        }
+
+        return result;
+}
+
+
+/**
+ * @brief Convert C DeleteAcls result response array to py list.
+ */
+static PyObject *
+Admin_c_DeleteAcls_result_responses_to_py (const rd_kafka_DeleteAcls_result_response_t **c_result_responses,
+                            size_t cnt) {
+        const rd_kafka_AclBinding_t **c_matching_acls;
+        size_t c_matching_acls_cnt;
+        PyObject *result;
+        size_t ti;
+
+        result = PyList_New(cnt);
+
+        for (ti = 0 ; ti < cnt ; ti++) {
+                PyObject *error;
+                const rd_kafka_error_t *c_error = rd_kafka_DeleteAcls_result_response_error(c_result_responses[ti]);
+
+                if (c_error) {
+                        error = KafkaError_new_or_None(
+                                rd_kafka_error_code(c_error),
+                                rd_kafka_error_name(c_error));
+                        PyList_SET_ITEM(result, ti, error);
+                } else {
+                        c_matching_acls = rd_kafka_DeleteAcls_result_response_matching_acls(
+                                                                        c_result_responses[ti],
+                                                                        &c_matching_acls_cnt);
+                        PyList_SET_ITEM(result, ti,
+                                Admin_c_AclBindings_to_py(c_matching_acls,c_matching_acls_cnt)
+                        );
+                }
+        }
+
+        return result;
+}
+
+/**
+ * @brief create_acls
+ */
+static PyObject *Admin_create_acls (Handle *self, PyObject *args, PyObject *kwargs) {
+        PyObject *acls_list, *future;
+        int cnt, i = 0;
+        struct Admin_options options = Admin_options_INITIALIZER;
+        PyObject *AclBinding_type = NULL;
+        rd_kafka_AdminOptions_t *c_options = NULL;
+        rd_kafka_AclBinding_t **c_objs = NULL;
+        CallState cs;
+        rd_kafka_queue_t *rkqu;
+        char errstr[128];
+
+        static char *kws[] = {"acls",
```

Comment:
```suggestion
        static const char *kws[] = {"acls",
```

Reply:
Do I change it in the other places too? (Admin_create_topics ...). Then I have to cast to  char ** when calling PyArg_ParseTupleAndKeywords

Reply:
Oh, okay, leave it as is then.